packages/opentelemetry-exporter-collector/src/types.ts

```diff
@@ -254,7 +253,32 @@ export namespace opentelemetryProto {
       status?: Status;
     }
 
-    export type Status = api.Status;
+    export interface Status {
+      /** The status code of this message. */
+      code: StatusCode;
+      /** A developer-facing error message. */
+      message?: string;
+    }
+
+    /**
+     * An enumeration of status codes.
+     * https://github.com/open-telemetry/opentelemetry-proto/blob/master/opentelemetry/proto/trace/v1/trace.proto#L304
+     */
+    export enum StatusCode {
```

Comment:
Can we make this (and all enums) const?

Again for better minification as during "compiliation" to JS, typescript converts all usages to the numeric value only.

The downside is that if you have code to convert "string" names back to an the enum value its not possible without adding your own extra "lookups"